pyvo/dal/query.py

```diff
@@ -376,9 +381,13 @@ def _findstatus(self, votable):
 
     def _findstatusinfo(self, infos):
         # this can be overridden to specialize for a particular DAL protocol
+        status = None
+        # return the last status to catch potential overflow or error sent
```

Comment:
This is interesting.  I had to go back and look at the DALI standard to see that this would work to the `OVERFLOW` case (i.e., that `OVERFLOW` would come last).